config/config.php

```diff
@@ -111,8 +116,12 @@
 		],
 	],
 	'daily' => [
+		'9.1' => [
+			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable10.zip',
+			'web' => 'https://docs.nextcloud.org/server/9/admin_manual/maintenance/manual_upgrade.html',
+		],
 		'9.0' => [
-			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest.zip',
+			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable9.zip',
```

Comment:
Ah ... I forgot that little 💎 ... we most likely don't want our users to run master ... stable branches should be fine here ;)